config/ember-try.js

```diff
@@ -59,20 +59,6 @@ module.exports = async function () {
           // FAIL_ON_DEPRECATION: true,
         },
       },
-      {
-        name: 'ember-default-with-jquery',
```

Comment:
Do we implicitly drop support for using with Ember jQuery? I guess there isn't a real risk to break jQuery support?

Reply:
No, I don't think so. This has been removed from the default blueprint. And it makes sense, since jQuery-support is now basically not a feature of Ember anymore. You can still use the user-land addon `@ember/jquery`, but I don't think we need to explicitly test anymore. Again, there is almost zero risk of breaking things, and we also don't test with any other possible JS library people might happen to use with Ember, so why jQuery...